tools/runners/Slang.py

```diff
@@ -18,6 +18,9 @@ def prepare_run_cb(self, tmp_dir, params):
         # a single compilation unit, so ask slang to do that.
         self.cmd += ['--single-unit']
 
+        if params['top_module'] != '':
```

Comment:
The top module is not explicitly defined for all the tests - because of this we have a helper that tries to guess the top module and use it, you may want to use it too:
https://github.com/SymbiFlow/sv-tests/blob/2aaf56c8ad4204129aa92c5e3fe044ae7e72051b/tools/runners/moore.py#L21-L24

Reply:
Thanks, but slang has no problem finding and instantiating the top modules based on the rules laid out in the LRM (all modules that are uninstantiated elsewhere, have no undefaulted parameters, etc).

This change is simply to limit the top modules considered when a test accidentally includes unused modules that should not be checked (which is common for the larger core tests that include unnecessary files).

Reply:
Sure, thanks for the explanation